0000-template.md

```diff
@@ -1,3 +1,14 @@
+<!---
+Copyright <fill in year> The Rust Project Developers. See the COPYRIGHT
```

Comment:
Yes, I've copied it over (I've literally typed `cp ../rust/LICENSE-* .` in my shell :p).

I don't think that saying "The Rust Developers" implies that this is some legal entity of some sort, but rather a collection of individuals of which each one continues to be copyright owner. Maybe I could remove the capitalization so that this becomes more obvious. What do you think?

Personally I think its better to say "The Rust Developers" than providing and maintaining a list of individuals who have made copyright protected changes to a work. Often these lists are terribly inaccurate.